rp2040-hal/src/dma/bidirectional.rs

```diff
@@ -0,0 +1,99 @@
+use super::{single_channel::SingleChannel, Pace, ReadTarget, WriteTarget};
```

Comment:
Honestly, why do we need the bidirectional config at all?
In your SPI example, you showed that the same can be achieved with two unidirectional configs as well.
This sounds to me like application level code, not HAL level code.

Reply:
> This sounds to me like application level code, not HAL level code.

I disagree. Having to resort to fabricating a SPI singleton and then throwing it away after the transaction, and having to do hacks to get both channels started simultaneously is not what something we want to push on our users.
This is enough to test my tests, but I would not recommend it as a general solution to the problem. We can do better.